test/fixtures/python/sum.invalid.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+
+num = eval(input("Enter a positive number: "
+if num < 0:
+   print("Non positive number"
```

Comment:
Instead of doing the same type of syntax error (missing paren), try using an invalid keyword or something else.

Reply:
Yep, I will.